src/views/design/ScriptDetail/index.tsx

```diff
@@ -310,25 +311,40 @@ const ScriptDetail = withStyles(styles)(
-                                        ? newScriptDetail.version.number : 0}
-                                    onChange={(e) => this.updateScript({
-                                        version: {
-                                            ...newScriptDetail.version,
-                                            number: parseInt(e.target.value, 10),
-                                        },
-                                    })}
-                                    type="number"
-                                    InputProps={{
-                                        disableUnderline: true,
-                                        inputProps: {
-                                            min: 0,
-                                        },
-                                    }}
-                                />
+                                <>
+                                    <TextInput
+                                        id="script-version"
+                                        label={translator('scripts.detail.side.script_version')}
+                                        value={newScriptDetail && newScriptDetail.version.number
+                                            ? newScriptDetail.version.number : 0}
+                                        onChange={(e) => this.updateScript({
+                                            version: {
+                                                ...newScriptDetail.version,
+                                                number: parseInt(e.target.value, 10),
+                                            },
+                                        })}
+                                        type="number"
+                                        InputProps={{
+                                            disableUnderline: true,
+                                            inputProps: {
+                                                min: 0,
+                                            },
+                                        }}
+                                    />
+                                    <TextInput
+                                        id="script-security-group"
+                                        label={translator('scripts.detail.side.script_security')}
+                                        value={newScriptDetail && newScriptDetail.securityGroupName
+                                            ? newScriptDetail.securityGroupName : ''}
+                                        onChange={(e) => this.updateScript({
+                                            securityGroupName: e.target.value,
+                                        })}
+                                        InputProps={{
+                                            disableUnderline: true,
+                                        }}
+                                        required
+                                    />
+                                </>
```

Comment:
when creating a new script and not providing a security group, I am still allowed to save the script. The same behavior (and implementation) as script name should be used. 
See 
```
error={requiredFieldsState.name.showError}
helperText={requiredFieldsState.name.showError && 'Scriptname is a required field'}
value={newScriptDetail && newScriptDetail.name
    ? newScriptDetail.name : ''}
onChange={(e) => this.updateScript({ name: e.target.value })}
required={this.isCreateScriptRoute()}
```

![Screenshot (78)](https://user-images.githubusercontent.com/6872974/104425077-82966000-5580-11eb-8c72-3d729fd7b2ce.png)